core/src/main/java/com/tdunning/math/stats/AVLTreeDigest.java

```diff
@@ -607,4 +634,46 @@ public static AVLTreeDigest fromBytes(ByteBuffer buf) {
         }
     }
 
+
+    private byte[] serializeRandomObj(Random r) {
+        ByteArrayOutputStream bos = new ByteArrayOutputStream();
+        try {
+            ObjectOutputStream oos = new ObjectOutputStream(bos);
+            oos.writeObject(r);
+            oos.flush();
+            byte [] data = bos.toByteArray();
+            bos.close();
+            oos.close();
+            return data;
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new RuntimeException("Cannot serialize random object");
+        }
+    }
+
+    private static Random deserializeRandomObj(byte [] bytes) {
+        ByteArrayInputStream bais = new ByteArrayInputStream(bytes);
+        try {
+            ObjectInputStream ois = new ObjectInputStream(bais);
+            Random r = (Random)ois.readObject();
```

Comment:
How is the object here constrained? I don't like creating objects of unknown type based on user data.

Reply:
`bytes` here isn't constrained - it was meant to cleanup the code in `fromBytes ()` a bit, but in retrospect, it doesn't really make sense to expose this function to the rest of the class. 

I'll make a note to move this into `fromBytes()` if serializing the `Random` object ends up being the path forward after some additional discussion